tests/components/homewizard/test_config_flow.py

```diff
@@ -385,6 +385,9 @@ async def test_reauth_flow(hass, aioclient_mock):
 
     device = get_mock_device()
     with patch(
+        "homeassistant.components.homewizard.async_setup_entry",
+        return_value=True,
+    ), patch(
```

Comment:
> It's probably good to patch the integration setup function here to avoid setting up the whole integration when finishing the reauth flow.

Like this?

Reply:
Yeah, looks good.